pkg/asset/manifests/openstack/cloudproviderconfig.go

```diff
@@ -65,9 +69,10 @@ func CloudProviderConfigSecret(cloud *clientconfig.Cloud) ([]byte, error) {
 	if cloud.RegionName != "" {
 		res.WriteString("region = " + strconv.Quote(cloud.RegionName) + "\n")
 	}
-	if cloud.CACertFile != "" {
-		res.WriteString("ca-file = " + strconv.Quote(cloud.CACertFile) + "\n")
```

Comment:
this looks like exactly what we want.  is this file already present somewhere in the cluster? Does the cloud-config configmap reference it?

Reply:
The problem is that currently `cloud.CACertFile` references a local file on the node that runs the installer. It should be changed to a well known path to where the configmap is mounted.

Reply:
It is also not supported in gophercloud right now I believe. @Fedosin  knows more than I do.

Reply:
But also, I second @mandre 's point. We dont want the user setting where the configmap is mounted based on the content of their clouds.yaml. It should be a static location.

Reply:
to clarify, the clouds.yaml supports these fields: https://github.com/gophercloud/utils/blob/cb37ab634bffc7c150154d1a60192e926d2eddbf/openstack/clientconfig/results.go#L18